### PR TITLE
Create list layout

### PR DESCRIPTION
**Pull request by German-Cobian**
Create the layout for to-do-list mostly from the index.js file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
...
+
+## Built With
+
+* Semantic HTML
+* JavaScript
+* Webpack
+* Github workflow
+* Linters(Lighthouse, stylelint, Webhint, eslint)
+
+
```
Comment (mricanho) [inline README.md:17]: - [ ] Add a section about how to run your app locally, just step by step instructions(e.g. how to clone, `npm install`, `npm run build` etc)

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @German-Cobian , 

You did a great job with the style and your JS code, just a minor detail on your README file and you are good to go! 🔥🔥 

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (German-Cobian) [reply README.md:27]: Implemented the changes suggested.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**